src/main/java/de/dennisguse/opentracks/TrackStoppedActivity.java

```diff
@@ -47,6 +48,21 @@ protected void onCreate(Bundle savedInstanceState) {
         ContentProviderUtils contentProviderUtils = new ContentProviderUtils(this);
         Track track = contentProviderUtils.getTrack(trackId);
 
+        //Temporary code to illustrate JSON serialization and loading of Track model
+        //TODO! - Remove
+        //Assignee - Jean Robatto
+
+        final String JSON_SERIALIZER_LOG_TAG = "JSONSerializerTest";
+
+        final String trackJSONString = track.toJSON();
+        Log.i(JSON_SERIALIZER_LOG_TAG, trackJSONString);
+
+        final Track trackCopy = JSONSerializable.fromJSON(trackJSONString, Track.class);
+        Log.i(JSON_SERIALIZER_LOG_TAG, trackCopy.toString());
+        Log.i(JSON_SERIALIZER_LOG_TAG, trackCopy.getName());
+
+        //End
```

Comment:
I did the same for phase 1. I wonder if we should create tests this phase or in phase2.

Reply:
I'd say leave them more towards the end given that we have limited functionality anyways. (IMO though, should be fine either way)